src/Behat/YiiExtension/Context/YiiContext.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Behat\YiiExtension\Context;
+
+class YiiContext implements YiiAwareContextInterface
```

Comment:
I'd like to keep that class. Anybody who wants to integrate this context to his feature contexts should imo use hooks, i.e. BeforeScenario. Having the class ready for integration seems natural.